src/components/TopBar.js

```diff
@@ -140,6 +149,21 @@ export default Utils.connectAtom(authStore, 'authState')(class TopBar extends Co
             ]),
             'See All Workspaces'
           ]),
+          div({ style: { marginTop: '2rem' } }, [
+            h(Clickable, {
+              style: styles.nav.miniItem,
+              onClick: () => this.setState({ showingSupportModal: true })
+            }, [
+              div({
+                style: {
```

Comment:
Can this just reference `styles.nav.icon`? I was curious to experiment as I'm learning about the code and I didn't notice any effect from the `color: colors.darkBlue[5]` part.